deepdrivemd/misc/OutlierDB.py

```diff
@@ -0,0 +1,49 @@
+import subprocess
```

Comment:
There should not be a `misc` module. This code can be added to the Agent implementation.

Reply:
The utilities there do not belong to any particular component.

aggregator_reader.py is used both by outlier search and ML to get the next batch of data to process.

OutlierDB.py is used by outlier search and, under more complicated policy than the current simplistic one, in MD (to get the next outlier). 

I would rather argue that utils.py belong to this directory since the amount of such utilities will likely to grow and will not be limited to a single file.

Reply:
Instead, perhaps we can move these to the deepdrivemd/data module and create an ADIOS API? This will really help to begin abstracting ADIOS out. Ideally we should have the ability to use the same class/interface to retrieve data whether the data is coming from an ADIOS stream or a file.

As an important note, the MD processes should not be responsible in anyway for the selection of outliers. All this logic should be contained by the Agent, this will be very important as we incorporate other Agent algorithms and MD engines.

Reply:
Sorry, this comment got lost for a few days :) 

Can we make a `deepdrivemd/data/stream/api.py` where we move the adios classes in the `misc` folder?

We can also move the OutlierDB into `deepdrivemd/data/stream/`. 

This will be a much better location, since we should not have a misc module as part of the package. These classes really belong to the data API.